src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -31,7 +34,11 @@ private fun compile(filename: String, outputVideoFile:String, generatePython: Bo
     }
     val stylesheet = Stylesheet(stylesheetPath, symbolTable)
 
-    val manimInstructions = VirtualMachine(abstractSyntaxTree, symbolTable, lineNodeMap, file.readLines(), stylesheet).runProgram()
+    val (exitStatus, manimInstructions) = VirtualMachine(abstractSyntaxTree, symbolTable, lineNodeMap, file.readLines(), stylesheet).runProgram()
+
+    if (exitStatus != ExitStatus.EXIT_SUCCESS) {
+        exitProcess(exitStatus.code)
```

Comment:
Referring to this line in my previous comment

Reply:
`Compiling...
Error detected during program execution. Animation could not be generated. 
Exit code: 300
Your program failed at line 2: Attempted to pop from empty stack x`

Reply:
That is what is produced

Reply:
Okay no worries then, looks good to go